src/main/java/lotto/domain/Analyzer.java

```diff
@@ -0,0 +1,71 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class Analyzer {
+
+    private final List<Integer> WIN_KEYS = Arrays.asList(3, 4, 5, 5, 6);
```

Comment:
`static`으로 선언해주면 어떨까요? [참고](https://itellyhood.tistory.com/40)

```suggestion
    private static final List<Integer> WIN_KEYS = Arrays.asList(3, 4, 5, 5, 6);
```

Reply:
블로그 참고하여 변경했습니다.!